Git/Pull Requests.md

```diff
@@ -0,0 +1,25 @@
+## Pull requests
+
+Definition and basic rules of a good PR can be found [here](https://infinum.com/handbook/dotnet/team-practices/new-team-members#prs).
+	
+There could also be additional pre-defined rules that every PR needs to satisfy before getting an approval (all required reviewers have approved the PR, all comments have been resolved, etc.). On top of that, all merge conflicts (concurrent changes of the same resources) should be resolved before the merge. 
+
+After merging, all changes from the source branch are available in (appended to) the target branch. Its pointer changes to point to the last commit of the pull request. Now, the source branch is obsolete and can be removed.
+
+### How to write a good PR description

```

Comment:
Just remembered this, should we maybe also add a section suggesting the usage of PR description templates?

Reply:
Did you mean this? 
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository